app/lib/controllers/session.js

```diff
@@ -5,14 +5,14 @@ const Api = require(`${paths.lib.models}/api`)
 
 const Session = function () {}
 
-Session.prototype.authorise = function (username, password, next) {
```

Comment:
It has been tested. I left the argument of the `authorise` function as `username` precisely for that reason. Took me a while to figure out what was breaking though!